landing-page/content/common/releases/release-notes.md

```diff
@@ -61,7 +57,106 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
+  * Change log tables can be queried without RowData serialization issues. [[\#3240](https://github.com/apache/iceberg/pull/3240)]
+  * Data overflow problem is fixed when writing time data of type `java.sql.Time` [[\#3740](https://github.com/apache/iceberg/pull/3740)]
+* **Hive**
+  * Hive metastore client retry logic is improved using `RetryingMetaStoreClient` [[\#3099](https://github.com/apache/iceberg/pull/3099)]
+  * Hive catalog can be initialized using a `null` Hadoop configuration instead of throwing exception [[\3252](https://github.com/apache/iceberg/pull/3252)]
+  * Table creation can succeed when some columns do not have comments instead of throwing exception [[\#3531](https://github.com/apache/iceberg/pull/3531)]
+  * `VectorizedOrcInputFormat` supports split offsets in `OrcTail` for better read performance  [[\#3748](https://github.com/apache/iceberg/pull/3748)]
+  * `FileIO` serialization can be disabled using Hadoop config `iceberg.mr.config.serialization.disabled` for better performance [[\#3752](https://github.com/apache/iceberg/pull/3752)]
+**Other notable changes:**
+
+* The community has finalized the long-term strategy of multi-version support of Spark, Flink and Hive. See [Multi-engine Support](/multi-engine-support) for more details.
+* The Iceberg Python module is renamed as [python_legacy](https://github.com/apache/iceberg/tree/master/python_legacy) [[\#3074](https://github.com/apache/iceberg/pull/3074)]. A [new Python module](https://github.com/apache/iceberg/tree/master/python) is under development to provide better user experience for the Python community. See the [Github Project](https://github.com/apache/iceberg/projects/7) for progress.
+* Iceberg starts to publish daily snapshot to the [Apache snapshot repository](https://repository.apache.org/content/groups/snapshots/org/apache/iceberg/) [[\#3353](https://github.com/apache/iceberg/pull/3353)] for developers that would like to consume the latest unreleased artifact.
+* Iceberg website is now managed by a separated repository [iceberg-docs](https://github.com/apache/iceberg-docs/) with a new layout. See [README](https://github.com/apache/iceberg-docs/blob/main/README.md) for how to make documentation contributions going forward.
+* An OpenAPI specification is developed for Iceberg catalog to prepare for a REST-based Iceberg catalog implementation [[\#3770](https://github.com/apache/iceberg/pull/3770)]
+* Dependency version upgrades:
+  * Gradle is upgraded to 7.3 [[\#3525](https://github.com/apache/iceberg/pull/3525)]
+  * ORC is upgraded to 1.7 [[\#3493](https://github.com/apache/iceberg/pull/3493)]
+  * Nessie is upgraded to 0.18 [[\#3890](https://github.com/apache/iceberg/pull/3890)]
+  * Arrow is upgraded to 6.0 [[\#3690](https://github.com/apache/iceberg/pull/3446)]
+  * Parquet is upgraded to 1.12.2 [[\#3551](https://github.com/apache/iceberg/pull/3551)]
+
+## Past releases
+
+### 0.12.0
```

Comment:
@samredai looks like 0.12.1 is not updated yet? Could you let me know when you do that, and I can rebase